src/SpecterWallet.ts

```diff
@@ -124,10 +126,10 @@ export class SpecterWallet {
 
   static isRebalanceNeeded({ lndBalance, onChain }) {
     // base number to calculate the different thresholds below
-    const lndHoldingBase = yamlConfig.rebalancing.lndHoldingBase
+    const lndHoldingBase = this.config.lndHoldingBase
```

Comment:
I just noticed that this wouldn't work with constructor injection. Converted the PR back to draft until this is addressed.

Reply:
From call with `bodymindarts`, we will ignore this for now since there is other work that might address this. For this **static** method we will stick with global configs